src/hm-admin/services/config/config.ts

```diff
@@ -0,0 +1,18 @@
+import {Injectable} from '@angular/core';
```

Comment:
filename should be `config.service.ts` or something alike, isn't it ? Test service creation with ng-cli to check that